src/test/ci/scoreboard.test.ts

```diff
@@ -99,22 +101,102 @@ describe("score/exp updating", () => {
     describe("position changes", () => {
         it("should return the correct ranking of every player", async () => {
             const players = {
-                "12345": new Player("", "12345", "", 2),
+                ohmiID: new Player("", "ohmiID", "", 2),
```

Comment:
Add tests for every player having different score, every player having same score